homeworks/vazgen_harutyunyan/Homework_3/Problem171.java

```diff
@@ -0,0 +1,17 @@
+//Calculate N!
+import java.util.Scanner;
+public class Problem171{
+    public static void main(String[] args) {
+        int n;
+        int factorial=1;
+        Scanner scanner = new Scanner(System.in);
+        n=scanner.nextInt();
+        if(n==0){
+        	System.out.print(factorial);
+        }
+        for(int i=1;i<=n;i++){
+        	factorial=factorial*i;
+        }
```

Comment:
this approach should have been used for all cases, you did not need to use arrays